table.go

```diff
@@ -1375,11 +1375,16 @@ func (t *TableBlock) writeRows(writer io.Writer, rowGroups []dynparquet.DynamicR
 	defer t.table.config.schema.PutWriter(w)
 	defer w.Close()
 
+	buffSize := 1
```

Comment:
What do you think of defaulting to something like `256`?

Reply:
yea I wasn't sure what a 'sane' default was which is why I left it at 1. But I'm happy to bump it to 256 if we want?